Makefile

```diff
@@ -29,6 +29,18 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+postman:
+	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
+	# and if needed a token.
+	# For example: make postman env=test token=<MY_TOKEN>
+	docker-compose -f docker-compose.test.yml run postman_test test/$(env).postman_environment.json --global-var "token=$(token)"
+
+
+postman-local:
+	docker-compose up -d
+	sleep 30
+	docker-compose -f docker-compose.test.yml run postman_test test/local.postman_environment.json
```

Comment:
To ensure we’re not using cached images or containers, can we build and force recreate (similar to the other make targets)?

Reply:
sure.  I will make this change shortly.